src/app/shared/shared.module.ts

```diff
@@ -1,10 +1,11 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { TranslateModule } from '@ngx-translate/core';
 import { SharedExampleComponent } from './components/shared-example/shared-example.component';
 
 @NgModule({
   declarations: [SharedExampleComponent],
-  imports: [CommonModule],
-  exports: [SharedExampleComponent],
+  imports: [CommonModule, TranslateModule.forChild({})],
```

Comment:
As you are already bootstrapping it on `app.module.ts`, you only need to import the `TranslateModule` here.

Reply:
It's required when lazy loading is being used.